### PR TITLE
Feature: DirectAdmin import

### LINKED ISSUE #3834: [Feature] DirectAdmin backup import
### Describe the feature or change in detail

I think it would be nice and helpful  to have an import function for DirectAdmin backups.
This will make it much easyer to migrate from DirectAdmin to Hestiacp.

Currently i'm working on a directadmin backup import script for Hestiacp based on the v-import-cpanel and [sk_da_importer](https://github.com/Skamasle/sk_da_importer). 
Hopefully this is helpfull for others, maybe we can add it to the project as v-import-directadmin in the bin folder when its working fine.

Are there requirements or criteria before we can add this to hestiacp?

### Would you like to sponsor this feature to have it implemented?

No

### PR DESCRIPTION
**Pull request by Aartsie**
Import DirectAdmin backup to a new user
What will be imported from DirectAdmin backups:

1. The user account information
2. The domains structure included all the files from `public_html` and/or `private_html`
3. All databases
    * **WARNING:** Only the first user with password will be imported because of HestiCP limits.
4. All the e-mail accounts including the e-mails.

Close:  #3834

### CODE REVIEW

File: bin/v-import-directadmin [hunk 1]
```diff
@@ -0,0 +1,311 @@
+#!/bin/bash
+# info: Import DirectAdmin backup to a new user
+#
+# example: v-import-directadmin /backup/backup.tar.gz
+#
+# Based on sk-da-importer
+# Credits: Maks Usmanov (skamasle) and contributors:
```
Comment (cafferata) [inline bin/v-import-directadmin]:
```suggestion
# Credits: Maks Usmanov (skamasle), Jaap Marcus (jaapmarcus) and contributors:
```

File: bin/v-import-directadmin [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+# Thanks to <https://github.com/Skamasle/sk_da_importer/graphs/contributors>
+
+# This script is provided whitout any warranty
+# Run at your own risk
+# Version 0.2
```
Comment (cafferata) [inline bin/v-import-directadmin]:
```suggestion
# Version 0.1
```

File: bin/v-import-directadmin [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+# This script restore backups from DA to Hestiacp
+# This script can restore Datases, databases user and passwords, mails and domains.
```
Comment (cafferata) [inline bin/v-import-directadmin]: This line can be removed, it's out of sync by `docs/docs/reference/cli.md`.

File: bin/v-import-directadmin (hunk 1 above)
Comment (Aartsie) [reply bin/v-import-directadmin]: Fixed by amend commit, thanks for the feedback!

File: bin/v-import-directadmin (hunk 3 above)
Comment (Aartsie) [reply bin/v-import-directadmin]: Fixed by amend commit, thanks for the feedback!

File: bin/v-import-directadmin (hunk 2 above)
Comment (Aartsie) [reply bin/v-import-directadmin]: Fixed by amend commit, thanks for the feedback!

File: bin/v-import-directadmin [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+tput setaf 2
+echo "Checking provided file..."
+tput sgr0
+if file $backup_file | grep -q -c "gzip compressed data,"; then
+	tput setaf 2
+	echo "OK - Gziped File"
+	tput sgr0
+	if [ ! -d /root/${tmp_dir} ]; then
+		echo "Creating tmp.."
+		mkdir /root/${tmp_dir}
```
Comment (jaapmarcus) [inline bin/v-import-directadmin]: We should create the temp folder in /backup/ instead /root/

Comment (Aartsie) [reply bin/v-import-directadmin]: Thank you for the response, i changed the temp folder from `/root/` to `/backup/`

**Comment by Aartsie** [comment]
@jaapmarcus Friendly reminder, I have changed the thing you requested. Do I need to do something else?
Or can i help you with something?

**Review by jaapmarcus — APPROVED** [review]

**Comment by jaapmarcus** [comment]
Just ran Prettier to fix the lint issues

**Pull request merged**